portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function validateForm() {
+  const sNumber = document.getElementById("number").value;
```

Comment:
Not sure what `sNumber` means here. This should probably be a more descriptive name, or add a comment to explain if you think this is the best label.

Reply:
re-named it to numberCommentsInput (meant it to signify the input number as a string when I was initially parsing it to an int, but then forgot to rename it after changing things around). Thanks!